scene/3d/mesh_instance_3d.cpp

```diff
@@ -493,6 +493,93 @@ void MeshInstance3D::create_debug_tangents() {
 	}
 }
 
+Ref<ArrayMesh> MeshInstance3D::create_mesh_from_blend_shape_mix() {
+	Ref<ArrayMesh> new_baked_mesh;
+	new_baked_mesh.instantiate();
+	Ref<ArrayMesh> source_mesh = get_mesh();
+	ERR_FAIL_NULL_V_MSG(source_mesh, new_baked_mesh, "No valid source mesh with blend shapes");
```

Comment:
The error should say it's not an ArrayMesh instance. the reason this line will fail is if it's not able to cast to ArrayMesh, not that it has or doesn't have blend shapes.